lib/eslint-tester.js

```diff
@@ -48,7 +48,10 @@ var assert = require("chai").assert,
     path = require("path"),
     merge = require("lodash.merge"),
     omit = require("lodash.omit"),
-    clone = require("lodash.clonedeep");
+    clone = require("lodash.clonedeep"),
+    validator = require("eslint/lib/config-validator"),
```

Comment:
We should export configvalidator so we don't have to do it this way, but OK for now.